app/views/steps/alternatives/lawyer_negotiation/edit.en.html.erb

```diff
@@ -1,58 +1,49 @@
-            </ul>
-          </div>
-
-          <div class="Cons">
-            <h2 class="ProsCons__heading ProsCons__heading--cons  heading-small gv-u-heading-small" aria-label="Cons of ">
-              <span data-block-name="ProsCons__heading--cons">Cons</span>
-            </h2>
-
-            <ul class="ProsCons__list ProsCons__list--cons">
-              <li class="ProsCons__item ProsCons__item--cons">can be a more expensive process</li>
-              <li class="ProsCons__item ProsCons__item--cons">you may feel you’re not in control</li>
-              <li class="ProsCons__item ProsCons__item--cons">can be seen as a confrontational approach</li>
-            </ul>
-          </div>
-        </div>
-
-        <p class="CallToAction">
-          <a class="CallToAction__link" href="https://helpwithchildarrangements.service.justice.gov.uk/lawyer-negotiation" rel="external" target="_blank">Find
-            out more about lawyer negotiation</a>
-        </p>
-
-        <%= step_form @form_object do |f| %>
-          <%= f.radio_button_fieldset :alternative_lawyer_negotiation, inline: true, choices: GenericYesNo.values %>
-          <%= f.continue_button %>
-        <% end %>
-      </div>
-    </div>
+    <p class="govuk-body">With lawyer negotiation you don’t deal directly with the person. You hire a lawyer to negotiate arrangements for you.</p>
+    <p class="govuk-body">You can still hire a lawyer to negotiate on your behalf even if the other person chooses not to use one.</p>
+    <p class="govuk-body">Lawyer negotiation is suitable for people who prefer not to meet because their relationship is still difficult, or because there’s a lack of trust.</p>
+
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :alternative_lawyer_negotiation, legend: { tag: 'span', size: 'm' } do %>
+        <%= f.govuk_radio_button :alternative_lawyer_negotiation, GenericYesNo::YES, link_errors: true %>
+        <%= f.govuk_radio_button :alternative_lawyer_negotiation, GenericYesNo::NO %>
+      <% end %>
+
+      <%= f.continue_button %>
+    <% end %>
   </div>
 
-  <div class=" column-one-third gv-o-grid-item gv-u-one-third">
+
+  <div class="govuk-grid-column-one-third">
+    <div class="app-sidebar-grey govuk-!-padding-3 govuk-!-margin-bottom-5">
+      <h3 class="govuk-heading-m">Pros</h3>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>your lawyer supports you throughout</li>
+        <li>quicker and less stressful than court</li>
+        <li>you don’t deal directly with the other parent</li>
+      </ul>
+
+      <h3 class="govuk-heading-m">Cons</h3>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>can be a more expensive process</li>
+        <li>you may feel you’re not in control</li>
+        <li>can be seen as a confrontational approach</li>
+      </ul>
+    </div>
+
+    <p class="govuk-body">
```

Comment:
I think this link is part of the main content (left side) right before the radios. It also happens in some of the other pages.